std/trig.ln

```diff
@@ -1,9 +1,11 @@
+// TODO: Probably change this from always unwrapping Result-wrapped values to passing along
+// result-wrapped errors
```

Comment:
Not probably, definitely. The usage of this version of `trig` in the test suite is terrible.